extensions/mongodb/pom.xml

```diff
@@ -47,19 +47,94 @@
                     </archive>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>com/hazelcast/jet/**</exclude>
+                    </excludes>
```

Comment:
Wouldn't it be clearer to just rename all tests to `*IT`?

Reply:
I have mixed feelings, I thought that keeping the original name will be less cluttered ;P https://github.com/hazelcast/hazelcast/pull/25744/commits/dbb3f108d58670686a2905b45fe4f98cd8f26a8a